Tests/Functional/TestValidLayouts.swift

```diff
@@ -50,6 +50,13 @@ class ValidLayoutsTestCase: XCTestCase {
         }
     }
 
+    func testEmptyPackageWithDirectTests() {
```

Comment:
add `testEmptyPackageWithDirectTests` in `ValidLayoutsTestCase`'s `XCTestCaseProvider` conformance